src/jti_background.js

```diff
@@ -77,6 +78,30 @@ function getData (callback) {
     });
 }
 
+// Return the "rate it now" flag
+function getRateStatus (callback) {
+    chrome.storage.sync.get(rateStatus, function (status) {
```

Comment:
If we use the one storage ID then this would become:
`chrome.storage.sync.get(StorageID, function (data) {
if (data[storageTogglesKey][rateStatus]) { ...`